### PR TITLE
Replace "new" tooltip with wonderblocks

### PR DESCRIPTION
**Pull request by nixterrimus**
## Summary:

Update the "new" tooltip to use wonderblocks.

Screen Shot 2022-09-27 at 2 11 23 PM

Issue: https://khanacademy.atlassian.net/browse/LP-11670

## Test plan:
- Make sure everything looks ok in the storybook

### CODE REVIEW

**Review by kevinbarabash — COMMENTED** [review]
Thanks for updating Perseus to use wonder-blocks for this.  It's great seeing so much code get deleted.

While I see a bunch of deleted files, I don't see any changes that look like the code switching over to the wonder-blocks tooltip.  Are there some changes in a different PR that I should be looking at or files that didn't get committed to this one?

I see that we're using the wonder-blocks `Popover` component for tooltips in perseus.  Usually this component is reserved for interactive content or non-textual content.  Is this something that's supported by Perseus content?  If so, it would be good to test some examples of this kind of content as part of the test plan to make sure it's being rendered as expected.

**Comment by nixterrimus** [comment]
@kevinbarabash The code you want to look at is in `packages/perseus/src/components/new-tooltip/new-tooltip.jsx`. I'm not sure why the Github diff doesn't show that that's the changed file. 🤷‍♀️

That file provides an API compatible interface to perseus with the new Wonderblocks code.

**Comment by kevinbarabash** [comment]
> @kevinbarabash The code you want to look at is in `packages/perseus/src/components/new-tooltip/new-tooltip.jsx`. I'm not sure why the Github diff doesn't show that that's the changed file. 🤷‍♀️
> 
> That file provides an API compatible interface to perseus with the new Wonderblocks code.

Thanks!  I'll review tomorrow morning. 🙂

**Review by kevinbarabash — CHANGES_REQUESTED** [review]
After looking at how `NewTooltip` is used in `HighlightTooltip`, a couple of thoughts came to mind:
- There isn't that much going on in the update `NewTooltip` and since it's only being used in `HighlightTooltip` could these two components be merged into one?
- The `label` prop in `HightlightTooltip` is a simple `string`.  This is the use case that `wonder-blocks-tooltip` was built for.  We should be using that instead of `wonder-blocks-popover` unless there is a really good reason.  I see that we want the styling to be a bit different.  There's already some code in `wonder-blocks-tooltip` to support styling it, it just hasn't be exposed in the prop types, see https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-tooltip/src/components/tooltip.js#L171.  I think we should add `style` to the prop types in `Tooltip` and use it instead of `Popover`.  It means that the tooltips won't have the `close` button anymore, but the old ones didn't have that either.  I'm happy to help out with the wonder-blocks changes.

**Comment by nixterrimus** [comment]
@kevinbarabash `wonder-blocks-tooltip` isn't built to programmatically opened and closed and that's one of the use cases for the tooltip. I agree that HightlightTooltip is the only EXPLICIT user but I have a hunch that this gets used in other places. It was previously it's on package so I want to keep it in existence and just let this be API compatible. There also are use cases where the close button is explicitly wanted and that's true of the previous design.

I'm in a bit of a time crunch, I leave for sabbatical soon and I want to make sure this is landed. Can we please land this and handle any further changes as further tickets?

File: packages/perseus/src/components/new-tooltip/README.md [hunk 1]
```diff
@@ -1,110 +1,3 @@
...
-Eventually, we'll roll out `NewTooltip` to all KA call sites, and upgrade it to support everything that the old `Tooltip` call sites need. So, no worries if you end up adding new `Tooltip` call sites; we'll help upgrade them soon! [For more details, see ADR #33.][adr-33]
-
-  [adr-33]: https://docs.google.com/document/d/1B4pMGWp344e96RmIbnsmUBiTbrizdbOvipe-9I_1iZQ/edit
-  [fixtures]: http://localhost:8081/react-sandbox/components/new-tooltip-package/new-tooltip.jsx.fixture.js
-  [react-portal]: https://reactjs.org/docs/portals.html
-  [reusable-components]: https://khanacademy.slack.com/messages/C4PE1QM5Y/
-
-Code-Owner: Frontend Infrastructure
-Crowdin-Category: _other_
+This is a wrapper around [Wonderblock's popover component](https://github.com/Khan/wonder-blocks). It is API compatiable with Perseus and remains for compatiability and legacy reasons. If you are anyone but the Perseus group at Khan Academy please use the wonderblocks component directly.
```
Comment (kevinbarabash) [inline packages/perseus/src/components/new-tooltip/README.md:3]: **praise:** Thanks for adding this warning here.

File: packages/perseus/src/components/new-tooltip/new-tooltip.jsx [hunk 2]
```diff
@@ -322,78 +94,62 @@ class NewTooltip extends React.PureComponent<Props, NewTooltipState> {
...
-                    toggleOnHover ? this._handleMouseEnter : undefined
-                }
-                onMouseLeave={
-                    toggleOnHover ? this._handleMouseLeave : undefined
+            <Popover
+                content={
+                    <PopoverContentCore
+                        color={this._popoverContentColor()}
+                        closeButtonVisible={dismissOnClickClose}
+                        style={{}}
```
Comment (kevinbarabash) [inline packages/perseus/src/components/new-tooltip/new-tooltip.jsx:127]: **nit (non-blocking):** this prop doesn't appear to be required.

**Review by kevinbarabash — COMMENTED** [review]
I didn't realize there was a time crunch.  I also didn't realize that we need to support the tooltips being shown programatically.  That's something that that wonder-blocks-tooltip should support.  I'll file a ticket for that as well as one for exposing a `style` prop on `Tooltip`.  Enjoy your sabbatical! 🎉

**Review by kevinbarabash — APPROVED** [review]
I forgot to hit "approve".

**Comment by kevinbarabash** [comment]
> I agree that HightlightTooltip is the only EXPLICIT user but I have a hunch that this gets used in other places. 

@nixterrimus I don't understand.  How could `NewTooltip` be used in other places with being explicitly used?

**Comment by kevinbarabash** [comment]
I've created https://khanacademy.atlassian.net/browse/WB-1410 and https://khanacademy.atlassian.net/browse/WB-1411 to track the required changes to `wonder-blocks-tooltip`.

**Pull request merged**